Luna_Streaming_2.8_Release_Notes.md

```diff
@@ -28,14 +27,26 @@ This release adds these features to the original Apache Pulsar 2.8.0 release:
  
 *Note:* The DataStax Luna Streaming Distribution is designed for Java 11. However, because the product releases Docker images, you do not need to install Java (8 or 11) in advance. Java 11 is bundled in the Docker image.
 
-## Source Code of Main Components
+### Source Code of Main Components
 
 The source code for these versions of Apache Pulsar and Apache BookKeeper code is in
    
    * https://github.com/datastax/pulsar
    * https://github.com/datastax/bookkeeper
 
-## Upgrade Considerations for Luna Streaming Distribution 2.8.0
+### Security
+Luna Streaming is developed with a great attention to security.
+
+Since it is based on Apache Pulsar&trade;, Luna Streaming Pulsar distribution gains all the security processes and requirements needed in the Apache Software Foundation&trade;, such as:
+* Peer to peer code reviews.
+* Clear process for handling possible security vulnerabilities.
+
+On top of that, Luna Streaming development processes are enhanced with additional security best practice for all the components involved:
+* Robust Continuous Integration and reproducible Release pipeline in a secure monitored environment. Such monitoring improves the overall security of the application. 
+* Reactivity to third-party vulnerabilities. Every components is scanned and monitored daily, ensuring quick fixes in case of vulnerability disclosure.
+* Regular docker images vulnerability scan on latest Luna Streaming releases, with a well-defined release procedure.
+
```

Comment:
Can we put a list of the components that passes the Sonatype scan at the time of the build? Something like:

At the time of the creation of this release, the following components were scanned by Sonatype (include link to tool) and did not have any critical (severity 8 or higher) security issues:

* component1
* component2
* etc

This list will be pretty static except when we add new components. Assuming we follow our process, this should always be true.

Reply:
I added a more generic sentence, otherwise every release line will contain duplicated contents. Since we have it on the release process, we grant this feature for every new release

Note that only the 'all' packages are supposed to be critical cve free (I specified it)